src/nupic/research/temporal_memory.py

```diff
@@ -102,11 +195,9 @@ def __init__(self,
     self._random = Random(seed)
 
     self.activeCells = set()
-    self.predictiveCells = set()
-    self.activeSegments = set()
     self.winnerCells = set()
```

Comment:
The `activeCells` and `winnerCells` can now be stored in a list or deque. We keep them sorted with the algorithm, so there doesn't have to be a runtime perf hit to keep them sorted.